odoo/addons/base/migrations/10.0.1.3/pre-migration.py

```diff
@@ -141,6 +141,33 @@ def ensure_country_state_id_on_existing_records(cr):
                 rows.append(row)
 
 
+def precreate_partner_fields(cr):
+    """ Emulate stored computed methods in a single SQL query """
+    cr.execute(
```

Comment:
As this is still 10.0, please use `add_fields` method in openupgradelib for proper adding of `ir.model.data` entry.

Reply:
That does not seem necessary in 10.0 anymore. With this code I get the required entries:
```
select * from ir_model_data where model = 'res.partner' AND name like '%commercial_company_name'or name like '%partner_share';
   id    | create_uid | create_date | write_date | write_uid | noupdate |              name               |         date_init          |        date_update         | module |      model      | res_id 
---------+------------+-------------+------------+-----------+----------+---------------------------------+----------------------------+----------------------------+--------+-----------------+--------
 2115118 |            |             |            |           |          | field_res_partner_partner_share | 2020-02-28 09:30:50.118351 | 2020-02-28 09:30:50.118351 | base   | ir.model.fields |  13503
 2115122 |            |             |            |           |          | field_res_users_partner_share   | 2020-02-28 09:30:50.118351 | 2020-02-28 09:30:50.118351 | base   | ir.model.fields |  13507
(2 rows)
```

Reply:
Uhm, you're right. Then I don't identify what is the root cause that led me to create `add_fields` method in openupgradelib... Any way, let's continue with this.